src/yolf.app.src

```diff
@@ -28,9 +28,8 @@
  [{description, "Erlang helpers and utility functions"},
   {vsn, "0.1.1"},
   {modules,
-   [
-    =MODULES=
-   ]},
+[]},
```

Comment:
I don't understand this change. It's a template, so if =MODULES= is empty it will end up with an empty array, no?

Reply:
The =MODULES= syntax is not supported in later erlang versions unfortunately